pkg/portal/bindata.go

```diff
@@ -10,6 +10,8 @@
 // ../../portal/v2/build/static/js/2.fc3140a1.chunk.js
```

Comment:
@facchettos would ask you to remove bindata from these.  Can we do so in a follow-up PR though? 

See #2119 as an example.

Reply:
Yep can do, will create an ADO ticket to do this